src/qtils/hex.hpp

```diff
@@ -17,23 +17,21 @@ namespace qtils {
    * @struct Hex
    * @brief Wrapper for hex encoding of byte sequences.
    */
-  struct Hex {
-    SpanAdl<const uint8_t> v;  ///< Underlying byte sequence to be hex-encoded
-  };
+  struct Hex : std::span<const uint8_t> {};
```

Comment:
If it's purpose is to avoid accidentally passing bytes which are not a hex encoding, shouldn't it be explicitly convertible from span (as opposed to implicit conversion)?

Reply:
No, this is wrapping type to link special formatter for some types.